python/cuml/_thirdparty/sklearn/utils/skl_dependencies.py

```diff
@@ -15,6 +15,53 @@
 from ....thirdparty_adapters import check_array
 
 
+def _get_feature_names(X):
+    """Get feature names from X.
+
+    Support for other array containers should place its implementation here.
+
+    Parameters
+    ----------
+    X : {ndarray, dataframe} of shape (n_samples, n_features)
+        Array container to extract feature names.
+
+        - dataframe : The columns will be considered to be feature
+          names. If the dataframe contains non-string feature names,
+          `None` is returned.
+        - All array containers will return `None`.
+
+    Returns
+    -------
+    names: ndarray or None
+        Feature names of `X`. Unrecognized array containers will return `None`.
+    """
+    feature_names = None
+
+    # extract feature names for support array containers
+    if hasattr(X, "columns"):
+        feature_names = np.asarray(X.columns, dtype=object)
```

Comment:
Tests are failing, because we never import `numpy` in this module. Please add the appropriate imports, similar to here:
https://github.com/rapidsai/cuml/blob/b8daf81a8c6f4d7c0fb4efe4b961afaa0dd0f3f1/python/cuml/_thirdparty/sklearn/preprocessing/_data.py#L49-L50